File: implementations/python/sdk/src/multiplexed_ai_sdk/contracts/control/execution_control_status.py
from enum import Enum


class AiSdkExecutionControlStatus(str, Enum):
    NONE = "None"
    RUNNING = "Running"
    PAUSING = "Pausing"
    PAUSED = "Paused"
    RESUMING = "Resuming"
    CANCELLING = "Cancelling"
    CANCELLED = "Cancelled"
    WAITING_FOR_INPUT = "WaitingForInput"

File: implementations/node/sdk/src/contracts/control/execution-control-status.ts
export const AI_SDK_EXECUTION_CONTROL_STATUSES = [
  "None",
  "Running",
  "Pausing",
  "Paused",
  "Resuming",
  "Cancelling",
  "Cancelled",
  "WaitingForInput",
] as const;

export type AiSdkExecutionControlStatus =
  (typeof AI_SDK_EXECUTION_CONTROL_STATUSES)[number];
